src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -140,26 +146,47 @@ protected override void ProcessRecord()
 
             try
             {
-                var parsedJson = JToken.Parse(Json);
+                var parsedJson = JsonNode.Parse(Json);
 
                 if (_jschema != null)
                 {
-                    var errorMessages = _jschema.Validate(parsedJson);
-                    if (errorMessages != null && errorMessages.Count != 0)
-                    {
-                        result = false;
-
-                        Exception exception = new(TestJsonCmdletStrings.InvalidJsonAgainstSchema);
-
-                        foreach (var message in errorMessages)
+                        var validationResults = _jschema.Validate(parsedJson, new ValidationOptions { OutputFormat = OutputFormat.Basic });
+                        result = validationResults.IsValid;
+                        if (!result)
                         {
-                            ErrorRecord errorRecord = new(exception, "InvalidJsonAgainstSchema", ErrorCategory.InvalidData, null);
-                            errorRecord.ErrorDetails = new ErrorDetails(message.ToString());
-                            WriteError(errorRecord);
+                            Exception exception = new(TestJsonCmdletStrings.InvalidJsonAgainstSchema);
```

Comment:
Since the error message is now a template, you need to use `string.Format()` so the template values get filled in

Reply:
There aren't any values on that line.

Reply:
In that case, you will need separate error strings to handle the two different cases